tools/packaging/common/istio-iptables.sh

```diff
@@ -116,6 +116,64 @@ function isIPv6() {
   done
   return 0
 }
+#
+# Maximum time to wait for an iptables call to succeed when populating the redirection rules.
+# During that time, iptables call will be attempted for every one second.
+#
+IPTABLES_WAIT=30
+#
+# The path to the iptables command.
+#
+# shellcheck disable=SC2230
+IPTABLES_CMD=$(which iptables)
+#
+# The path to the ip6tables command.
+#
+# shellcheck disable=SC2230
+IP6TABLES_CMD=$(which ip6tables)
+#
+# If true, retries iptables & ip6tables commands in case they fails.
+#
+IPTABLES_RETRY=false
```

Comment:
Ahh... I see what you did there.  Took me a min.  Hopefully with the tighter change block future patch applies will be smooth.

Reply:
Yeah, that change was necessary to make shellcheck pass. It doesn't like what looks like using a function before it's defined.